app/controllers/surveyor/responses_controller.rb

```diff
@@ -132,7 +132,13 @@ def destroy
   end
 
   def complete
-    @survey = Response.find(params[:response_id]).survey
+    @response = Response.find(params[:response_id])
+    if @response.respondable_id && @response.respondable.organization.survey_completion_alerts
+      ### sent emails to all relevant super users for an organization of which survey_completion_alerts is true
+      @response.respondable.organization.all_super_users.each do |su|
+        SurveyNotification.service_survey_completed(@response, @response.respondable, su).deliver
```

Comment:
Could you change this to use `deliver_later` so that the jobs are queued in the background?